<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>500.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="8" column="1">
            <widget class="QSpinBox" name="blue_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>blue</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QPushButton" name="agregar_final_pushButton">
             <property name="text">
              <string>Agregar Final</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="origen_y_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="id_lineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>origen x</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="distancia_spinBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>origen y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="origen_x_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="velocidad_spinBox">
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>destino x </string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="green_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>velocidad</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="red_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>red</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>green</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>distancia</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="2">
            <widget class="QPushButton" name="plainText_id_pushButton">
             <property name="text">
              <string>ID (ASENDENTE)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>destino y</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="2">
            <widget class="QPushButton" name="plainText_distancia_pushButton">
             <property name="text">
              <string>DISTANCIA (DESCENDENTE)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="destino_x_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QPushButton" name="agregar_inicio_pushButton">
             <property name="text">
              <string>Agregar Inicio</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QPushButton" name="mostar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>id</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="destino_y_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <widget class="QPushButton" name="plainText_velocidad_pushButton">
             <property name="text">
              <string>VELOCIDAD (ASCENDENTE)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="salida"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string> Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="table_id_pushButton">
          <property name="text">
           <string>ID (ASENDENTE)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="table_distancia_pushButton">
          <property name="text">
           <string>DISTANCIA (DESCENDENTE)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="table_velocidad_pushButton">
          <property name="text">
           <string>VELOCIDAD (ASCENDENTE)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Dibujo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar_pushButton">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar_pushButton">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
